deploy/template.yaml

```
@@ -1087,6 +1087,9 @@ Resources:
         IntegrationResponses:
           - StatusCode: "200"
             SelectionPattern: "200"
+            ResponseParameters:
+              integration.response.header.Content-Type: "application/did+json"
```

Think this should be `method.` rather than `integration.`
```suggestion
              method.response.header.Content-Type: "application/did+json"
```

---

If you find that this doesn't work give this a go:
```yaml
ResponseModels:
  application/did+json: !Ref DidJsonModel

# In the resources section
DidJsonModel:
    Type: AWS::ApiGateway::Model
    Properties:
      ContentType: 'application/did+json'
      RestApiId: !Ref RestApiGateway
      Schema: {}
```